Dockerfile.ubi

```diff
@@ -78,7 +78,8 @@ RUN INSTALL_PKGS="gcc \
 COPY . .
 
 # removing CA bundle of httpclient gem
-RUN find / -name httpclient -type d -exec find {} -name *.pem -type f -delete \;
+RUN find / -name 'httpclient-*' -type d -exec find {} -name '*.pem' -type f -delete \; && \
+    find / -name 'httpclient-*' -type d -exec find {} -name '*.key' -type f -delete \;
```

Comment:
Can this be added to the `RUN` block above for bundle install, to consolidate our container build layers?